yeat/workflow/Paired

```diff
@@ -58,3 +58,23 @@ rule unicycler:
         unicycler -1 {input.read1} -2 {input.read2} -t {threads} -o {params.outdir} {params.extra_args}
         ln -s assembly.fasta {output.contigs}
         """
+
+
+rule penguin:
+    output:
+        contigs="analysis/{sample}/paired/{label}/penguin/contigs.fasta"
+    input:
+        read1="seq/downsample/{sample}/paired/{sample}.R1.fq.gz",
+        read2="seq/downsample/{sample}/paired/{sample}.R2.fq.gz"
+    threads: 128
+    params:
+        outdir="analysis/{sample}/paired/{label}/penguin",
+        extra_args=lambda wildcards: config["assemblies"][wildcards.label].extra_args
+    shell:
+        """
+        penguin guided_nuclassemble {input.read1} {input.read2} {params.outdir}/unpolished_contigs.fasta {params.outdir} --threads {threads} {params.extra_args}
```

Comment:
PenguiN is primarily used for short-reads only. No support for long read assembly. Possibly could attempt to assembly long reads by using the single-end mode with PenguiN. I have not tested this personally.